server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/GeoDistanceAggregationBuilder.java

```diff
@@ -229,15 +229,17 @@ public GeoDistanceAggregationBuilder(String name, GeoPoint origin) {
 
     private GeoDistanceAggregationBuilder(String name, GeoPoint origin,
                                           InternalRange.Factory<InternalGeoDistance.Bucket, InternalGeoDistance> rangeFactory) {
-        super(name, rangeFactory.getValueType());
+        // TODO: Not sure if we need to capture rangeFactory.getValueType() here...
```

Comment:
Same as with the `AbstractRangeBuilder`,  it wasn't clear to me if we need to be doing something with the range type information from the range factory.